ipl-auction-react/src/control/outLinedTextField.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./outLinedTextField.css";
```

Comment:
Use css modules

Reply:
I will add CSS modules and update them in code.